server/controllers/api/siteMetadataController/index.js

```diff
@@ -33,11 +35,9 @@ const SiteMetadataController = {
               }
             )
             if (!isParticipantInSiteMetadata) {
-              await SiteMetadataModel.upsert(
-                appDb,
-                { study },
-                { $addToSet: { participants: participant } }
-              )
+              await SiteMetadataModel.upsert(appDb, { study }, undefined, {
```

Comment:
Why is this undefined now? I don't usually see that passed explicitly as an argument, which isn't necessarily bad but does make me curious.

Reply:
Can we make the third argument the fourth argument so it can be optional?

Reply:
as of mongo 5.0 the atomic operators $set and $addToSet can now take an empty object, so the code from before was from version 4.4 where an empty object results in an error. 
[0d3fb2a](https://github.com/AMP-SCZ/dpdash/pull/723/commits/0d3fb2a533cd17a1c2038482525486b45d8b1ac2)